### DISCUSSION TITLE
Volume control

### DISCUSSION THREAD

**Discussion by sebasgra39** [hassio-addons/addon-spotify-connect#82] [Ideas]
Hi Frenck,

is it possible to acceess the volume control via an automation or script at the moment? 

I'd love to have this kind of feature for multiple reasons. First, I'm building an ESP Home-based board with physical buttons and triggers for different purposes. An "old school" rotary encoder for my spotify volume would fit great there. Second, in different cases an automation that lowers the volume (e.g. connected to door bell) would be great too. Please think about this! 

BR
Sebastian

**Comment by sebasgra39** [comment]
Hey Guys,

any idea or update?

BR
Sebastian

**Comment by ovizii** [comment]
With the **spotify integration** you can easily control the volume.
